plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -160,6 +160,8 @@ class read_only {
       account_resource_limit     cpu_limit;
       int64_t                    ram_usage = 0;
 
+      uint32_t                   subjective_cpu_bill = 0;
```

Comment:
This needs to be optional so that old cleos will work with new nodeos. Please test with new cleos <-> old nodeos and old cleos <-> new nodeos.

Reply:
Thanks. Will change and test different versions of cleos.

Reply:
I have tested with different versions of nodeos and cleos. cleos worked in all combinations.

Reply:
I would think this would make more sense as a field in its own structure similar to `net_limit` and `cpu_limit`. This is basically the `used` field in `account_resource_limit`. And eventually, not now, we will want the `max` and `available` too.

Reply:
Thanks. I will do that.